<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>functionWindow</class>
 <widget class="QMainWindow" name="functionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>题包列表</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="packFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>720</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>720</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget_packs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>750</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pack_OK">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>240</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>330</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>下一页</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_last">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>420</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>上一页</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_page">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>150</y>
       <width>113</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string>第 1 页</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_pack_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>751</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <zorder>label_page</zorder>
    <zorder>listWidget_packs</zorder>
    <zorder>pushButton_pack_OK</zorder>
    <zorder>pushButton_next</zorder>
    <zorder>pushButton_last</zorder>
    <zorder>progressBar_pack_frame</zorder>
   </widget>
   <widget class="QFrame" name="exerciseFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>720</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_questionlist_OK">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>240</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_questionlist_Back">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>330</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_questions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>750</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Beginpack">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>420</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>开始题包</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_deadline">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>150</y>
       <width>140</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;截止日期：&lt;/p&gt;&lt;p&gt;无限制</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_question_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>751</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <zorder>listWidget_questions</zorder>
    <zorder>pushButton_questionlist_OK</zorder>
    <zorder>pushButton_questionlist_Back</zorder>
    <zorder>pushButton_Beginpack</zorder>
    <zorder>label_deadline</zorder>
    <zorder>progressBar_question_frame</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
